src/components/People.js

```diff
@@ -0,0 +1,71 @@
+import "../App.css";
+
+import React, { Component } from 'react'
+
+class People extends Component {
+    constructor(){
+        super()
+        this.state={
+            characterData:[],
+            userInput: ""
+        }
```

Comment:
```suggestion
        this.state={
            characterData:[],
            userInput: "",
            selectedCharacter: ""
        }
```
you need this additional piece of state called `selectedCharacter` to keep track of when the user has officially submitted a search. Otherwise, you have no way of knowing when the display should go from showing **null** to either **"Not Found"** or the **character's info**.